agent-packaging/packaging-scripts/build_packages_deb.sh

```diff
@@ -15,22 +15,21 @@
 
 set -e
 
-function exit_error
-{
-  echo "build failed"
-  exit 1
-}
+BASE_DIR=${PWD}
 
-trap exit_error ERR
+echo "started build..."
+GCS_PATH="osconfig-agent-package"
+BASE_REPO="GoogleCloudPlatform"
```

Comment:
if this is hardcoded here i'm unsure why we even need it as a variable

Reply:
removed the variable